kusto/ingest/internal/gzip/gzip.go

```diff
@@ -69,7 +69,9 @@ func (s *Streamer) run() {
 		defer zw.Close()
 		defer zw.Flush()
 
-		_, err := io.Copy(zw, s.userInput)
+		amount, err := io.Copy(zw, s.userInput)
+		atomic.AddInt64(&s.size, int64(amount))
```

Comment:
Why Add?
Can this be run on multiple streams?

Reply:
Yea i guess you are right - it was before add because we did the chunking but here the copy does it for us